tests/test_fft_forward/LICENSE.rst

```diff
@@ -0,0 +1,84 @@
+*******************************
```

Comment:
Please delete this license file and every other license file that exists in a sub-directory of the tests directory.  Because we do not release tests separately from the repository, these license files are not necessary.  Having duplicate license files where not necessary increases the risk of differences creeping in.